README.md

```diff
@@ -15,10 +15,10 @@ The Splunk distribution of [OpenTelemetry JS](https://github.com/open-telemetry/
 
 This Splunk distribution comes with the following defaults:
 
-- [W3C context propagation](https://www.w3.org/TR/trace-context).
+- [W3C tracecontext and baggage propagation](https://www.w3.org/TR/trace-context).
 - [OTLP exporter](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-exporter-collector-proto)
-  configured to send spans to a locally running [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
-  (`http://localhost:55681/v1/traces`).
+  configured to send spans to a locally running [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) over HTTP
+  (default endpoint: `http://localhost:55681/v1/traces`).
 - Unlimited default limits for [configuration options](#trace-configuration) to
```

Comment:
@Rauno56 Our JS distro sends data to the collector using OTLP over HTTP? If that's the case, the Access Token is mandatory and the default endpoint is always listening on :55681 instead of the usual gRCP port. Is that right? That's significantly different from others distros.